spec/models/forms/form_name_spec.rb

```diff
@@ -102,7 +94,7 @@
         let(:et) { true }
 
         it 'returns hash with form_name being ET' do
```

Comment:
Is this needed? Since the ET is being removed, the `et` variable will never be `true` (line 76) correct? Maybe remove the whole context block?